notebooks/010_test_load.ipynb

```diff
@@ -42,7 +42,7 @@
     "# and an initial 3m water temperature value, which should be a \n",
     "# 7-day mean\n",
     "\n",
-    "ds = xr.open_dataset(os.path.join(dirs['param_files'],'temperature_insitu_3m_norkyst800_barentswatch_closest_20060101-20220920_trend.nc'))\n",
+    "ds = xr.open_dataset(os.path.join(forecast_clarify.dirs['param_files'],'temperature_insitu_3m_norkyst800_barentswatch_closest_20060101-20220920_trend.nc'))\n",
```

Comment:
Since `dirs` is imported from `forecast_clarfiy.config`, shouldn't this be `dirs` instead of `forecast_clarfiy.dirs`? For me, the current version does not work.

Reply:
It depends on how you would like it to be used. In the `__init__.py` we expose the pieces of code we want to be accessable. Currently it is:

```python
# __init__.py
from forecast_clarify.main import *
from forecast_clarify.clarify_persistence_package import *
from forecast_clarify.config import *

__version__ = "0.1.0a"
```

This means when you do `import forecast_clarify`, you get access to all code on that level. If you want it to be `dirs` you can do `from forecast_clarify import dirs`. So it is really just a preference thing.